epfast-annotations/src/main/java/com/exactpro/epfast/converter/Converter.java

```diff
@@ -0,0 +1,158 @@
+package com.exactpro.epfast.converter;
+
+import java.math.BigDecimal;
+import java.math.BigInteger;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Converter {
+    private static final List<Character> HEX_ARRAY =
+        "0123456789ABCDEF".chars().mapToObj(e -> (char) e).collect(Collectors.toList());
+
+    //Decimal conversions
+    public Integer decimaltoInt32(BigDecimal decimalValue) {
```

Comment:
You got me wrong. From types should be FAST types, To types should be normal Java types

Reply:
return type for this method should be int. We assume, that `null` will be never passed to any of these methods as parameter